src/modules/esl-utils/misc/object/compare.ts

```diff
@@ -12,3 +12,12 @@ export function deepCompare(obj1: any, obj2: any): boolean {
   }
   return false;
 }
```

Comment:
Proposal: rename `deepCompare` rename to `isEqual`
(non breaking change with current alias)

Feel free to share your reaction for that proposal with emojy or comment